Sources/Basics/Collections/IdentifiableSet.swift

```diff
@@ -42,6 +42,10 @@ public struct IdentifiableSet<Element: Identifiable>: Collection {
         Index(storageIndex: self.storage.elements.endIndex)
     }
 
+    public var values: OrderedDictionary<Element.ID, Element>.Values {
```

Comment:
Would `some Collection<Element>` or `some Sequence<Element>` work better here? Then we don't have to commit to the exact collection type in a public API.

Reply:
Switched to `some Sequence<Element>`!